src/components/PeopleTable/PeopleTabel.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import { Person } from '../../types';
+
+type Props = {
+  people: Person[] | undefined,
+  slugPerson: string | undefined,
+};
+
+export const PeopleTable: React.FC<Props> = ({ people, slugPerson }) => {
+  return (
+    <>
+      {people && people.length > 0 && (
+        <table
+          data-cy="peopleTable"
+          className="table is-striped is-hoverable is-narrow is-fullwidth"
+        >
+          <thead>
+            <tr>
+              <th>Name</th>
+              <th>Sex</th>
+              <th>Born</th>
+              <th>Died</th>
+              <th>Mother</th>
+              <th>Father</th>
+            </tr>
+          </thead>
+          <tbody>
+            {people.map((person) => {
+              const {
+                name,
+                sex,
+                born,
+                died,
+                fatherName,
+                motherName,
+                slug,
+              } = person;
+              const mother = people.find(
+                (child) => (child.motherName === person.name),
+              ) || { slug: '' };
+              const father = people.find(
+                (child) => (child.fatherName === person.name),
+              ) || { slug: '' };
```

Comment:
```suggestion
              const mother = people.find(
                (child) => (child.motherName === person.name),
              )
              const father = people.find(
                (child) => (child.fatherName === person.name),
              );
```
if mother or father wil be not fouded, it will be undefined.
and see comment below in jsx how to change and simplify conditions